base_stage_state/views/stage_state.xml

```diff
@@ -0,0 +1,386 @@
+<?xml version="1.0"?>
```

Comment:
Is this just a placeholder for the planned UI? Looks like a direct copy of project KanBan?